sql/hive/src/test/scala/org/apache/spark/sql/hive/HiveSparkSubmitSuite.scala

```diff
@@ -163,6 +165,7 @@ class HiveSparkSubmitSuite
   }
 
   test("SPARK-9757 Persist Parquet relation with decimal column") {
+    assume(!SystemUtils.isJavaVersionAtLeast(JavaVersion.JAVA_9))
```

Comment:
We don't have `spark.sql.hive.metastore.version=0.12` in this test case. Could you add more explanation about this into the PR description? 
> spark.sql.hive.metastore.version of these tests is lower than 2.0

Reply:
https://github.com/apache/spark/blob/d7b7a1ac02659940f0e70e5cfb23c2bc34fb8775/sql/hive/src/test/scala/org/apache/spark/sql/hive/HiveSparkSubmitSuite.scala#L674-L685

Reply:
Thanks. Got it!